src/Profile.js

```diff
@@ -5,7 +5,7 @@ class Profile extends Component {
   render() {
     /* TODO: render information about logged in user */
     /* STRETCH TODO: if no logged in user then redirect home */
-    return <p>Profile page coming soon</p>
+    return <p>{this.props.data}</p>
```

Comment:
- In App.js we passed props to profile of ` <Profile data={this.state.user}/>`
- So inside this component, we most likely want to make use of Bootstrap Card or something similar so we can show the user's name and email etc... 
- For now I think this is ok but we will for sure edit this as we move through the lab
- Really great job on all of this 💯